admin_guide/tcp_ingress_external_ports.adoc

```diff
@@ -0,0 +1,99 @@
+[[admin-guide-unique-external-ips-ingress-traffic]]
+= Assigning Unique External IPs for Ingress Traffic
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+Cluster administrators can assign a unique IP address to a service that, if
+routed correctly, enables external traffic to reach the ports of that service.
+This external IP assignment allows you to expose an external port, enabling
+ingress traffic (TCP/UDP) routing to a definable backend. This is much simpler
+than trying to manage the port space of a limited number of shared IP addresses.
+
+There is support for both automatic and manual assignment of IP addresses, each
+address is guaranteed to be assigned to a limit of not more than one service.
+This ensures that each service can simply direct traffic from its chosen ports,
+regardless of the ports selected by other services.
+
+[[unique-external-ips-ingress-traffic-restrictions]]
+== Restrictions
+
+To use `*IngressIP*`, you can:
+
+- Select an IP address with the existing `*ExternalIP*` ingress.
+- Have an IP address assigned from a pool. In this case, {product-title} implements a bare-metal version of the LoadBalancer service type and assigns IP addresses to the services.
```

Comment:
Consider changing all references to 'bare-metal' with 'non-cloud' or something similar, since the ingress controller will work on a vm-based cluster (technically non-cloud).